File: uri/1 - Basics/1015/Main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var x1, y1, x2, y2 float64

	fmt.Scanf("%f %f", &x1, &y1)
	fmt.Scanf("%f %f", &x2, &y2)

	fmt.Printf("%.4f\n", math.Sqrt(math.Pow((x2-x1), 2)+math.Pow((y2-y1), 2)))
}

File: uri/1 - Basics/1015/Main.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var l1 = lines[0].split(' ').map(parseFloat)
var l2 = lines[1].split(' ').map(parseFloat)
var x1 = l1[0]
var y1 = l1[1]
var x2 = l2[0]
var y2 = l2[1]
const distance = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2))
console.log(distance.toFixed(4))